### ISSUE TITLE
allowbackup issue

### ISSUE THREAD

**Issue by fengyanliang** [Ashok-Varma/BottomNavigation#92]
I used MVP clean framework, it suggest to set allowbackup=false. But this project required to set allowbackup=true.

**Comment by kjf3006** [comment]
Please remove android:allowBackup="true" from manifest to fix this issue.

@fengyanliang a workaround until then would be: 

```
<manifest xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools">

<application
        tools:replace="android:allowBackup"
        android:allowBackup="false" />
```

**Comment by Ashok-Varma** [comment]
fixed in 1.4.3 thanks for the support
